### PR TITLE
TiDB Cloud: data service custom domain feature

### PR DESCRIPTION
**Pull request by clark1013**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@clark1013 https://github.com/clark1013/docs/pull/1 PTAL

**Review by heibaijian — APPROVED** [review]

File: tidb-cloud/data-service-custom-domain.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+TiDB Cloud Data Service provides a default domain `<region>.data.tidbcloud.com` to access each Data App's endpoints. For enhanced personalization and flexibility, you can configure a custom domain for your Data App instead of using the default domain.
+
+This document describes how to manage custom domains in your Data App.
+
+## Before you begin
+
+Before configuring a custom domain for your Data App, note the following:
+
+- Custom domain requests exclusively support HTTPS for security. Once you successfully configure a custom domain, a Let's Encrypt certificate is automatically applied.
```
Comment (hfxsd) [inline tidb-cloud/data-service-custom-domain.md:16]:
```suggestion
- Custom domain requests exclusively support HTTPS for security. Once you successfully configure a custom domain, a "Let's Encrypt" certificate is automatically applied.
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/data-service-custom-domain.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+TiDB Cloud Data Service provides a default domain `<region>.data.tidbcloud.com` to access each Data App's endpoints. For enhanced personalization and flexibility, you can configure a custom domain for your Data App instead of using the default domain.
+
+This document describes how to manage custom domains in your Data App.
+
+## Before you begin
+
+Before configuring a custom domain for your Data App, note the following:
+
+- Custom domain requests exclusively support HTTPS for security. Once you successfully configure a custom domain, a "Let's Encrypt" certificate is automatically applied.
+- Your custom domain must be unique with the TiDB Cloud Data Service.
```
Comment (Oreoxmt) [inline tidb-cloud/data-service-custom-domain.md:17]:
```suggestion
- Your custom domain must be unique within the TiDB Cloud Data Service.
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**